### ISSUE TITLE
Error-"Excel finds unreadable content in file... do you want to recover contents of this workbook"?

### ISSUE THREAD

**Issue by pathfindercb** [mk-j/PHP_XLSXWriter#103]
Excel 2007 displays above message loading even simple files created by class. After recovering the file seems fine. Once recovered workbook is saved the sheet1.xml is much simpler without some of the settings (like zoomscalepagelayoutview and with strings stored in sharedstrings.xml (see sheet1orig.xml and sheet1.recovered).
Interesting that WPS Spreadsheet loads without error message.
Is this an issue with Excel2007 that is fixed in Excel 2010+?

[UserAddresses (3).xlsx](https://github.com/mk-j/PHP_XLSXWriter/files/894872/UserAddresses.3.xlsx)

[sharedStrings.xml.txt](https://github.com/mk-j/PHP_XLSXWriter/files/894915/sharedStrings.xml.txt)

[sheet1recovered.xml.txt](https://github.com/mk-j/PHP_XLSXWriter/files/894868/sheet1recovered.xml.txt)
[sheet1orig.xml.txt](https://github.com/mk-j/PHP_XLSXWriter/files/894869/sheet1orig.xml.txt)

**Comment by pathfindercb** [comment]
Just tested with Excel 2013 and Excel 2016 and getting the same error opening the file
![excel 2013 error](https://cloud.githubusercontent.com/assets/24591549/24720513/52e7eed2-1a0b-11e7-9c65-845c5142574c.jpg)

**Comment by mk-j** [comment]
What version are you using? The latest version no longer uses: "sharedstrings.xml"

**Comment by pathfindercb** [comment]
Sorry, I had the right version but problem with my webserver and wrapper class. Works great now!. I branched and will add the ability to set column widths for your review...

**Comment by patrickporteous** [comment]
Hi, I'm having this same problem with one column of data that I'm attempting to export.  I have 27 other columns that are working fine.  I've been able to determine that it's something in the string causing the error by exploding the string into an array and isolating the part of the string that's causing the error.  In looking at the data/string being returned in Access, I haven't been able to determine exactly what's causing the error.  Are there any characters that are known to cause problems in this class when creating the .xlsx file?  I've tried  str_replace & preg_replace to remove spaces and dashes and also htmlspecialchars but still no fix.  My error_logs don't show an error in the coding so I'm guessing it's an issue with Excel and how the file is constructed.
